governor/src/state/direct.rs

```diff
@@ -3,6 +3,8 @@
 //! Rate limiters based on these types are constructed with
 //! [the `RateLimiter` constructors](../struct.RateLimiter.html#direct-in-memory-rate-limiters---constructors)
 
+#![allow(unused_imports)]
```

Comment:
I think this is unnecessary (and extremely broad!). The warning about unused imports for the `use future::*` below can be fixed by removing the entire line - I'm not sure why nightly insists there are unused imports when nothing is imported, but eh... let's remove that line below.

Reply:
Removing